autogpt_platform/backend/backend/data/graph.py

```diff
@@ -156,33 +192,14 @@ def ending_nodes(self) -> list[Node]:
             v for v in self.nodes if isinstance(get_block(v.block_id), AgentOutputBlock)
         ]
 
-    @property
-    def subgraph_map(self) -> dict[str, str]:
-        """
-        Returns a mapping of node_id to subgraph_id.
-        A node in the main graph will be mapped to the graph's id.
-        """
-        subgraph_map = {
-            node_id: subgraph_id
-            for subgraph_id, node_ids in self.subgraphs.items()
-            for node_id in node_ids
-        }
-        subgraph_map.update(
-            {node.id: self.id for node in self.nodes if node.id not in subgraph_map}
-        )
-        return subgraph_map
-
```

Comment:
is this intentional @majdyz ?

Reply:
> It was intentional, when creatablegraph was introduced there was a breakage because now not all the object creation is constructed from from_db. I forgot the exact locations where it's breaking but you can revert the code before this commit and see the breakage on graph execution unrelated to the GitHub triggers. 
> 
> And since the subgraph feature is unused, I'm removing the feature entirely.

Reply:
I'm removing the functionality here:
https://github.com/Significant-Gravitas/AutoGPT/pull/8526/files